functional-tests.sh

```diff
@@ -202,22 +212,33 @@ function test_make_bucket()
 
     start_time=$(get_time)
     bucket_name="mc-test-bucket-$RANDOM"
-    assert_log_failure "$start_time" "${FUNCNAME[0]}" mc_cmd mb "${SERVER_ALIAS}/${bucket_name}"
-    assert_log_failure "$start_time" "${FUNCNAME[0]}" mc_cmd rm "${SERVER_ALIAS}/${bucket_name}"
+    assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd mb "${SERVER_ALIAS}/${bucket_name}"
+    assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd rm "${SERVER_ALIAS}/${bucket_name}"
+
+    log_success "$start_time" "${FUNCNAME[0]}"
+}
+
+function test_make_bucket_error() {
+    show "${FUNCNAME[0]}"
+
+    start_time=$(get_time)
+    bucket_name="Mc-test-bucket-$RANDOM"
```

Comment:
Could you add more invalidation than single `M`?

Reply:
> Could you add more invalidation than single M?

Done